src/corefx/System.Globalization.Native/CMakeLists.txt

```diff
@@ -6,12 +6,25 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 add_definitions(-DPIC=1)
 add_definitions(-DBIT64=1)
 
-find_library(ICUUC NAMES icuuc)
+find_library(ICUUC NAMES icuuc PATHS "/usr/local/opt/icu4c/lib")
```

Comment:
Do you think it would be a good idea to define this path in a variable, and then use the variable in the 3 places?